### DISCUSSION TITLE
how to find IMTEXT class

### DISCUSSION THREAD

**Discussion by DELAGREEN** [mozman/ezdxf#1138] [Q&A]
Hello,

I have been working on modified DWG drawings, specifically with the AutoCAD client. Using your library, I am trying to generate a list of all classes. In the AutoCAD client, I can see the IMTEXT class, but when I attempt to print the list from the library, the IMTEXT class is not included. As a result, I cannot access the IMTEXT class to extract the text for the technical requirements of the drawing.

I am reaching out to you as the creator of the library, hoping that your expertise and experience might provide some insights. Could you please assist me with this issue?

Thank you for your help.

**Comment by mozman** [comment]
There is no documentation for an `IMTEXT` entity in the DXF reference, nor have I seen any.

**Reply by DELAGREEN** [reply]
What do you think, is it worth digging in the direction of oda-converter-file??

**Reply by mozman** [reply]
I don't understand your question.

**Reply by DELAGREEN** [reply]
Sorry, I didn't think about it.

Is it worth checking out the ODA-converter-file? 
Or is it beyond help at this point?

**Reply by mozman** [reply]
In what context?
